### ISSUE TITLE
Failure of essential tasks under default solver

### ISSUE THREAD

**Issue by JHL-452b** [SysBioChalmers/Human-GEM#821]
#### Current behavior:

As we discussed in #820, if there is no problem with recently updated content, two bug in current metabolic tasks:
- Essential tasks failed under default solver(`GLPK`), but not with `gurobi`;
- Failed tasks could not get any feedback.

#### Proposed changes:

Modify the line 38 in the testMetabolicTasks function as below
```matlab
setRavenSolver('gurobi')
% change the third parameter to `true` to printOutput.
verificationTaskReport=checkTasks(model,[],true,true,false,verificationTasks);
```

**Comment by edkerk** [comment]
Regarding the solver in #820 is suggested that RAVEN should automatically swap solvers. This would not be preferred behaviour of native RAVEN functions, to swap mid-function to a different solver that might not be installed on the user's computer. A better behaviour is indeed to set the preferred RAVEN solver before running `testMetabolicTasks`. However, that would make this task depend on gurobi, which again might not always be installed/functional.

Irrespective, this tasks shouldn't fail with `glpk`. I looked into the RAVEN code, and it seems like some `glpk` parameters are incorrectly set, which cause the FBA here to time out.

This is now solved in https://github.com/SysBioChalmers/RAVEN/pull/539, which will soon end up in RAVEN 2.9.1 release.

The `printOutput` parameter should still be resolved in this repo.
